thirdeye-ui/src/app/platform/components/page-v1/page-contents-grid-v1/page-contents-card-v1/page-contents-card-v1.interfaces.ts

```diff
@@ -12,11 +12,13 @@
  * See the License for the specific language governing permissions and limitations under
  * the License.
  */
+import type { CardProps } from "@material-ui/core/Card";
 import { ReactNode } from "react";
 
 export interface PageContentsCardV1Props {
     fullHeight?: boolean;
     disablePadding?: boolean;
     className?: string;
     children?: ReactNode;
+    otherProps?: CardProps;
```

Comment:
Make sure to add this to https://github.com/startreedata/startree-ui/blob/master/projects/platform-ui/src/components/page-v1/page-contents-grid-v1/page-contents-card-v1/page-contents-card-v1.interfaces.ts as well

Reply:
Create a ticket and this is a chore task

Reply:
Created a new ticket [TE-1063](https://cortexdata.atlassian.net/browse/TE-1063) and moved this commit to a new PR #782 with a proper `chore` commit message